<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>SettingPage</class>
 <widget class="QWidget" name="SettingPage">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>400</width>
    <height>300</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>SettingPage</string>
  </property>
  <layout class="QVBoxLayout" name="verticalLayout">
   <property name="leftMargin">
    <number>20</number>
   </property>
   <property name="topMargin">
    <number>20</number>
   </property>
   <property name="rightMargin">
    <number>0</number>
   </property>
   <property name="bottomMargin">
    <number>0</number>
   </property>
   <item>
    <widget class="HorizonNavigation" name="horizonNavigation" native="true"/>
   </item>
   <item>
    <widget class="QStackedWidget" name="settingStackedPage">
     <widget class="::DefaultSetting" name="defaultSetting"/>
     <widget class="::AccountSetting" name="accountSetting"/>
     <widget class="::AdvanceSetting" name="advanceSetting"/>
    </widget>
   </item>
  </layout>
 </widget>
 <customwidgets>
  <customwidget>
   <class>HorizonNavigation</class>
   <extends>QWidget</extends>
   <header>SUI/HorizonNavigation.h</header>
  </customwidget>
  <customwidget>
   <class>::DefaultSetting</class>
   <extends>QWidget</extends>
   <header>ClientUi/Setting/DefaultSetting.h</header>
   <container>1</container>
  </customwidget>
  <customwidget>
   <class>::AccountSetting</class>
   <extends>QWidget</extends>
   <header>ClientUi/Setting/AccountSetting.h</header>
   <container>1</container>
  </customwidget>
  <customwidget>
   <class>::AdvanceSetting</class>
   <extends>QWidget</extends>
   <header>ClientUi/Setting/AdvanceSetting.h</header>
   <container>1</container>
  </customwidget>
 </customwidgets>
 <resources/>
 <connections/>
</ui>
